source/jormungandr/tests/bragi_autocomplete_tests.py

```diff
@@ -263,3 +264,40 @@ def http_get(url, *args, **kwargs):
             return MockResponse({}, 200, '')
         with mock.patch('requests.get', http_get) as mock_method:
             self.query_region('places?q=bob&type[]=stop_point&type[]=address')
+
+
+@dataset({"main_routing_test": {}})
+class TestBragiShape(AbstractTestFixture):
+    def test_places_for_user_with_shape(self):
+        """
+        Test that with a shape on user, it is correctly posted
+        """
+        with user_set(app, "test_user_with_shape"):
+            def http_get(url, *args, **kwargs):
+                assert False
+
+            def http_post(url, *args, **kwargs):
+                json = kwargs.pop('json')
+                assert json.get('geometry')
+                return MockResponse({}, 200, '{}')
+
+            with mock.patch('requests.get', http_get) as mock_method:
+                with mock.patch('requests.post', http_post) as mock_method:
+                    self.query('v1/coverage/main_routing_test/places?q=toto&_autocomplete=bragi')
```

Comment:
can you add a boolean to assert (or a custom `mock` function) that the http_post method as bee used ?